cmd/influxd/main.go

```diff
@@ -78,11 +74,13 @@ func main() {
 	case "":
 		execRun(args)
 	case "backup":
+		setGoMaxProcs()
```

Comment:
That's a good question for @benbjohnson.  Not sure if more procs helps out with any of the system packages we depend on.